### ISSUE TITLE
MD057 (relative links): LSP diagnostic being misplaced

### ISSUE THREAD

**Issue by chrisgrieser** [rvben/rumdl#226]
The lsp diagnostic for `MD057` seems to be placed at the wrong location. (using `rumdl` via `nvim`, if that matters)

<img width="1259" height="418" alt="Image" src="https://github.com/user-attachments/assets/4214b800-8c88-4883-bb57-c2ab8ec2b2a8" />

Using `rumdl` as cli does result in the correct location, though:

```bash
rumdl check
?? General/Lorem Ipsum.md:3:3: [MD057] Relative link 'ffffff' does not exist
```

**Comment by rvben** [comment]
Fixed in v0.0.200. Thanks for reporting @chrisgrieser!
